tests/dummy/app/pods/view/renderers/models/autocomplete.js

```diff
@@ -0,0 +1,14 @@
+export default {
+  properties: {
+    cookies: {
+      enum: [
```

Comment:
Maybe using an `enum` isn't the best choice for demo. Since you're trying to show that `autocomplete` is a little different from select. Using an `enum` the value is validated again the `enum`. For `autocomplete` I believe free text is allowed which would fail validation if that text wasn't in`enum`.

Reply:
In fact, you might want to provide some protection against using `enum` for `autocomplete`. Do you agree?

Reply:
Well, it's just a different style of dropdown, i see no reason to block using enum. It's up to the end user. The example under examples for autocomplete/autocomplete uses a query. I could switch that example to use a query

Reply:
It's not just style that's different though. With `select` you have to choose from the list of available options. With `autocomplete` you don't have to. This means the value can optionally come from the dropdown list and that behavior breaks the meaning of `enum`.

So I can understand not wanting to alter the behavior to intentionally block `enum` but at the very least, lets not encourage people to use it with `enum`.

Reply:
yeah, in the documentation I will list what scenario is ideal for autocomplete. IE, wanting the user to type to search for options